stable/restore/files/nuorestore

```diff
@@ -181,102 +168,66 @@ if ! isUrl "$restore_source"; then
    fi
 fi
 
-if isRestoreRequestSupported; then
```

Comment:
Feel free to skim the rest of this file. The following changes are done:

- The code in the if block just has an indentation change.
- The code in the else block is removed.